pythonlib/amlrealtimeai/client.py

```diff
@@ -22,7 +23,7 @@
 
 class PredictionClient:
 
-    def __init__(self, address: str, port: int, use_ssl:bool = False, access_token:str = ""):
+    def __init__(self, address: str, port: int, use_ssl:bool = False, access_token:str = "", channel_shutdown_timeout:timedelta = timedelta(minutes=2)):
```

Comment:
So I know we've really only seen this bug on Ubuntu, but do we want similar behavior with dotnet core too?